Src/FluentAssertions/NumericAssertionsExtensions.cs

```diff
@@ -1181,6 +1196,11 @@ public static class NumericAssertionsExtensions
             double unexpectedValue, double precision, string because = "",
             params object[] becauseArgs)
         {
+            if (double.IsNaN(unexpectedValue))
```

Comment:
This is the only of the eight extension methods on `NullableNumericAssertions` where the `IsNaN` check is added.

Reply:
Unless, I'm missing the point, all of them end up calling the same methods, which are protected by `IsNan`.

Reply:
Roger.
Then this one is not necessary, as `unexpectedValue` is checked inside the call to `NotBeApproximately` on line 1212.

What triggered me was that I saw 5, i.e. an odd number of added calls to `IsNaN`, either float or double had too many/few calls to `IsNaN`.